tensorbay/opendataset/BDD100K_MOT2020/loader.py

```diff
@@ -248,25 +251,27 @@ def _get_mots_data(
         )
         labeled_multipolygons.append(labeled_multipolygon)
 
-    mask_path = os.path.join(mask_subdir, f"{stem}.png")
+    mask_paths = [os.path.join(mask_subdir, f"{stem}.png") for mask_subdir in mask_subdirs]
     mask_info = _save_and_get_mask_info(
         os.path.join(original_mask_subdir, f"{stem}.png"),
-        mask_path,
-        os.path.join(mask_subdir, f"{stem}.json"),
+        mask_paths,
+        os.path.join(mask_subdirs[1], f"{stem}.json"),
     )
-    ins_mask = InstanceMask(mask_path)
+    ins_mask = InstanceMask(mask_paths[1])
     ins_mask.all_attributes = mask_info["all_attributes"]
 
     label = data.label
     label.multi_polygon = labeled_multipolygons
+    label.semantic_mask = SemanticMask(mask_paths[0])
```

Comment:
use list[0] for semantic path and list[1] for instance path is hard to understand,
shall we use tuple or dict or two variables rather than list?